### PR TITLE
app-office/calcurse: Updated calcurse to version 4.6.0

### PR DESCRIPTION
**Pull request by niyumard**
Package-Manager: Portage-3.0.4, Repoman-3.0.1

### CODE REVIEW

File: app-office/calcurse/calcurse-4.6.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (juippis) [inline app-office/calcurse/calcurse-4.6.0.ebuild:4]: Always aim to use the latest available EAPI which is 7 currently.

File: app-office/calcurse/calcurse-4.6.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+PYTHON_COMPAT=( python3_{6..8} )
```
Comment (juippis) [inline app-office/calcurse/calcurse-4.6.0.ebuild:5]: Looking at deps, python-3.9 should also work.

File: app-office/calcurse/calcurse-4.6.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+DEPEND="
+	${RDEPEND}"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-4.6.0-tinfo.patch
+)
+
+# Most tests fail.
+RESTRICT="test"
```
Comment (juippis) [inline app-office/calcurse/calcurse-4.6.0.ebuild:31]: Any chance you'd try to fix these failing tests?
https://dev.gentoo.org/~mgorny/python-guide/test.html there are many pointers to fixing test issues here.

File: app-office/calcurse/files/calcurse-4.6.0-tinfo.patch [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+@@ -51,6 +51,9 @@
+   LIBS="$LIBS -l$available_ncurses"
+ ])
+ 
++AC_SEARCH_LIBS([cbreak], [tinfow tinfo],,
++    AC_MSG_ERROR([Cannot find a library providing cbreak]))
++
+ AC_CHECK_HEADERS([pthread.h], [
+     AC_CHECK_LIB(pthread, pthread_create, [
+ 	LIBS="$LIBS -pthread"
```
Comment (juippis) [inline app-office/calcurse/files/calcurse-4.6.0-tinfo.patch:21]: You don't have to replicate the patch file if it's identical. You can call `PATCHES=( "${FILESDIR}"/name-of-the-previous.patch )`.

**Review by juippis — COMMENTED** [review]
Then you have to properly sign-off your commit, please see the linked document about it:
https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin

**Comment by niyumard** [comment]
A new version is released and I'll try writing an ebuild for that with your great comments in mind, thank you very much.

**Pull request closed without merging**